docs/sphinx-docs/source/conf.py

```diff
@@ -79,7 +79,7 @@
 # The short X.Y version.
 version = '5.0'
 # The full version, including alpha/beta/rc tags.
-release = '5.0.5-alpha.1'
+release = '5.0.5a1'
```

Comment:
The existing docstring is no longer valid and should be changed. Adding a link to the StrictVersion rules should suffice.

http://epydoc.sourceforge.net/stdlib/distutils.version.StrictVersion-class.html